src/graph/graphcontext.c

```diff
@@ -389,7 +389,8 @@ int GraphContext_DeleteIndex(GraphContext *gc, const char *label,
 							 const char *field, IndexType type) {
 	ASSERT(gc != NULL);
 	ASSERT(label != NULL);
-	ASSERT(field != NULL);
+	// field may be null when dropping a full-text index
+	ASSERT(type == IDX_FULLTEXT || field != NULL);
```

Comment:
I think we should push this check further down the line to `Schema_RemoveIndex` 

also you might want to refine the assert:
```
if(type == IDX_FULLTEXT) ASSERT (field == NULL);
if(type != IDX_FULLTEXT) ASSERT (field != NULL);
```

would it be possible to enable debug assert when building with DEBUG?
`make DEBUG=1`